src/erf.jl

```diff
@@ -20,3 +19,53 @@ function erfc(a::Interval{T}) where T
     isempty(a) && return a
     @round( erfc(a.hi), erfc(a.lo) )
 end
+
+for f in (:erfinv, :erfcinv)
+
+    @eval function($f)(x::BigFloat, r::RoundingMode)
+        setrounding(BigFloat, r) do
```

Comment:
I don't understand this definition.
These rounded functions should be using the result of `_erfinv`.

Reply:
e.g. `erfinv(x, ::RoundDown)` = `_erfinv(Interval(x)).lo`

Reply:
Oh! Thanks for clearing that up.

Reply:
Hmm... I am not sure I understood something though. Why `Interval(x)`? Shouldn't it be just `x`?
```julia
erfinv(x, r::RoundingMode{:Down}) = _erfinv(x).lo
erfinv(x, r::RoundingMode{:Up}) = _erfinv(x).hi
```

Using `Interval(x)` creates an interval of an interval and I get an error from `roots`.